apps/dg/english.lproj/function_strings.json

```diff
@@ -396,7 +396,7 @@
       {
         "name": "filter",
         "type": "filter",
-        "description": "A true-false expression that limits the cases considered."
+        "description": "A true-false expression that limits the cases considered. Note that the string needs to be inside quotes."
```

Comment:
There is no mention of a string here, so the note doesn't apply. True of a number of the descriptions that follow this one.